### ISSUE TITLE
attention_mask for pretrain in language_modeling_model.py 

### ISSUE THREAD

**Issue by Hustcw** [ThilinaRajapakse/simpletransformers#1334]
**Describe the bug**
attention_mask not set in pretraining stage. Is it a bug ? Without attention_mask, padding tokens will feed into selfattention

**Expected behavior**
dataset should prepare attention_mask for mlm task

**Screenshots**
![image](https://user-images.githubusercontent.com/23634386/147919509-fd7241d5-2a51-426a-a568-5db989e86a8f.png)
